src/python/py/models/builder.py

```diff
@@ -1166,8 +1181,14 @@ def has_final_norm(self, module, model):
         return hf_norm or hf_final_layernorm or gguf_final_norm
 
     def make_attention_mask_reformatting(self):
+        if self.enable_cuda_graph:
+            # ORT does not allow nodes to be placed on mulitple execution providers
+            # with cuda graph enabled. Thus the attention mask is deprecated and the
+            # subgraph is replaced with seqlens_k and total_seq_len as the raw
+            # inputs to GroupQueryAttention.
+            return
```

Comment:
so all nodes need to be run on CUDA to enable cuda graph?

Reply:
Yes